config/label/attribute.yaml

```
@@ -1,5 +1,5 @@
 pedestrian_state:
-    sitting: [pedestrian_state.sitting_lying_down, pedestrian_state.sitting]
+    sitting: [pedestrian_state.sitting_lying_down, pedestrian_state.siting_lying_down, pedestrian_state.sitting]
```

Is this typo intentional? : `siting`

---

I added this mapping since this typo was included in the existing T4 dataset v1.0 or 2.0. 
If it feels odd to add this in the main branch, I can remove it. What do you think?

---

Thanks! I thought adding a comment would be sufficient. How about this?
```suggestion
    # The word `siting` is a typo included intentionally since the existing T4dataset also had the same typo.
    sitting: [pedestrian_state.sitting_lying_down, pedestrian_state.siting_lying_down, pedestrian_state.sitting]
```

---

Sounds nice. Thank you!